build.sbt

```diff
@@ -114,7 +194,7 @@ def sharedNativeImageSettings = Seq(
       )
 
 lazy val `proxy-core` = (project in file("proxy/core"))
-  .enablePlugins(JavaAppPackaging, DockerPlugin, AkkaGrpcPlugin, JavaAgent, AssemblyPlugin, GraalVMNativeImagePlugin)
+  .enablePlugins(DockerPlugin, AkkaGrpcPlugin, JavaAgent, AssemblyPlugin, GraalVMPlugin)
```

Comment:
We should be able to remove the AssemblyPlugin too now, right?

Reply:
The tck still uses it by default, ie in CI, I think.

Reply:
That's true. I guess if we have a Docker-client in CI we could launch it using that?

Reply:
Yes, though building the native image takes a *lot* of resources, on my laptop it takes approx 5 minutes with close to 100% of all cores/hyperthreads being used concurrently (4 cores x 2 hyperthreads). Also, it needs 2GB of RAM (it oomed when I limited it to 2GB). I wouldn't be surprised if travis can't do it. Perhaps we could allow still building the non native image docker images and use that.